packages/arcgis-rest-portal/src/util/SearchQueryBuilder.ts

```diff
@@ -3,10 +3,16 @@ import { IParamBuilder, warn } from "@esri/arcgis-rest-request";
  * `SearchQueryBuilder` can be used to construct the `q` param for [`searchItems`](/arcgis-rest-js/api/portal/searchItems#searchItems-search) or [`searchGroups`](/arcgis-rest-js/api/portal/searchGroups#searchGroups-search). By chaining methods, it helps build complex search queries.
  *
  * ```js
+ * const startDate = new Date("2020-01-01").valueOf();
+ * const endDate = new Date(2020-09-01).valueOf();
```

Comment:
```suggestion
 * const startDate = new Date("2020-01-01");
 * const endDate = new Date(2020-09-01);
```
based on the code below, i was under the impression that users could pass a plain old instance of `Date()`.
is that not the case?
https://github.com/Esri/arcgis-rest-js/blob/8d18794ec30f346b3af1269207c1953dd475bc9f/packages/arcgis-rest-portal/src/util/SearchQueryBuilder.ts#L308-L310

Reply:
@jgravois dunno.  i can circle back after trying plain old dates.  i do know Date().valueOf() works.

Reply:
yup. it definitely works and i'd recommend that we document it that way, because its more straightforward.

https://codepen.io/jgravois/pen/WNwZeez